Orange/preprocess/fss.py

```diff
@@ -66,7 +66,10 @@ def __call__(self, data):
 
         if not isinstance(data.domain.class_var, method.class_type):
             raise ValueError(("Scoring method {} requires a class variable " +
-                              "of type {}.").format(type(method), method.class_type))
+                              "of type {}.").format(
+                method.__name__ if hasattr(method, '__name__') else method,
```

Comment:
Ugh, I missed that method can be a class or an instance. Nice catch.
In that case:
```(method if type(method) == type else type(method)).__name__```